docs/how-to-guides/streams/manage-streams-lab.md

```diff
@@ -0,0 +1,148 @@
+---
+id: manage-streams-lab
+title: How to manage data streams from Memgraph Lab
+sidebar_label: Manage data streams from Memgraph Lab
+---
+
+If you prefer to use GUI, you can connect to data streams by using a wizard in
+the **Stream** section of Memgraph Lab. If you prefere writing commands, you can
+[manage streams with queries](/how-to-guides/streams/manage-streams.md). 
+
+If you need a Kafka stream to play around with, we've provided some at [Awesome
+Data Stream](https://awesomedata.stream/)! 
+
+[![Related - Reference Guide](https://img.shields.io/static/v1?label=Related&message=Reference%20Guide&color=yellow&style=for-the-badge)](/reference-guide/streams/overview.md) [![Related -
+Tutorial](https://img.shields.io/static/v1?label=Related&message=Tutorial&color=008a00&style=for-the-badge)](/tutorials/graph-stream-processing-with-kafka.md)
+
+## How to add a stream?
+
+To add a stream in Memgraph Lab: 
+
+1. Switch to **Streams** and **Add New Stream**.
+2. Choose stream type, enter a stream name, server address, and topics you want to subscribe to.
+3. Go to the **Next Step**.
+4. Click on **Edit** (pencil icon) to modify the *Consumer Group*, *Batch
+   Interval* or *Batch Size*.
+
+If you are trying to connect to MovieLens Kafka data stream from the [Awesome Data
+Stream](https://awesomedata.stream/#/movielens), the stream configuration should
+look like this:
+
+<img src={require('../../data/tutorials/create-stream-lab/creating-stream-movielens.png').default}className={"imgBorder"}/>
+
+Once the basic configuration is finished, you need to define a transformation
+module and attach it to the stream. 
+
+## How to add a transformation module?
+
+A transformation module is a set of user-defined transformation procedures
+written in [C](/reference-guide/streams/transformation-modules/api/c-api.md) or
+[Python](/reference-guide/streams/transformation-modules/api/python-api.md) that
+act on data received from a streaming source. Transformation procedures instruct
+Memgraph on how to transform the incoming messages to consume them correctly.
+
+At the moment, you can only develop Python transformation modules directly from
+Memgraph Lab. 

```

Comment:
This sentence may be misinterpreted - Python transformation modules can be developed only from Memgraph Lab. I know that this is well written, but I would emphasize Lab first and _only_ before Python: At the moment, in Memgraph Lab, you can develop transformation modules only in Python.

Reply:
i've rewritten this whole section